client/src/holdings/HoldingsList.tsx

```diff
@@ -218,6 +223,7 @@ export function HoldingsList({user, fxRates}: IProps) {
               </Button>
             </Tooltip>
           )}
+          <div>{Boolean(isRefreshing) && <CircularProgress />}</div>
```

Comment:
No need for the outer div, and can you make the size of it 25 as in the example in AddDividend